### ISSUE TITLE
JS редирект

### ISSUE THREAD

**Issue by komancher** [dvygolov/BinomoCloaker#13]
Можно ли через данную клоаку вставлять JS интеграции в сторонние сайты?

**Comment by dvygolov** [comment]
Нет, пока что такой функции не делал. В будущем, вероятно, добавлю, но не
обещаю.
